gateways/authorizenet/authorizenet.go

```diff
@@ -134,6 +135,24 @@ func (client *AuthorizeNetClient) RefundWithContext(ctx context.Context, request
 		return nil, err
 	}
 
+	var transactionDetailsResponse *sleet.TransactionDetailsResponse
+	if request.Options != nil && request.Options[sleet.GooglePayTokenOption] != nil {
+		var boolValue bool
+		boolValue, _ = strconv.ParseBool(request.Options[sleet.GooglePayTokenOption].(string))
+		if boolValue {
+			transactionDetailsResponse, err = client.GetTransactionDetails(&sleet.TransactionDetailsRequest{
+				TransactionReference: request.TransactionReference,
```

Comment:
Curious: so this would be original transaction?

Reply:
Yes